src/floats.jl

```diff
@@ -349,15 +349,15 @@ end
                     code |= INVALID
                     x = f === nothing ? x : nothing
                 else
-                    x, code = scale(T, FT, digits, -signed(frac), neg, code, f)
+                    x, code = scale(T, FT, digits, -signed(frac), neg, code, ndigits, f)
```

Comment:
Do we also need to actually _add_ to `ndigits` in `parsefrac`/`parseexp`? Or are we just looking for the `ndigits` accumulated from `parsedigits`?

Reply:
Oh we need to add those from parsefrac, just not those from parseexp. And I think we need to skip leading and trailing zeros.